notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<p>This cell has a lot of information but is a little hard to follow and has some typos. I think in this case it is ok to just tell the user which PyTorch version they need rather than explaining the whole naming scheme (which seems pretty complicated).  Here is my attempt to simplify (I recommend replacing this whole cell, but feel free to adjust):</p><p> </p><pre class="ql-syntax" spellcheck="false">You need to specify two Python packages when you create your version resource. One of these is the package containing <code>model.py</code> that you uploaded to Cloud Storage in a previous step. The other is a package containing the version of PyTorch that you need.

Google Cloud provides a collection of PyTorch packages in the <code>gs://cloud-ai-pytorch</code> Cloud Storage bucket. These packages are mirrored from the [official builds](https://download.pytorch.org/whl/cpu/torch_stable.html).

For this tutorial, use <code>gs://cloud-ai-pytorch/torch-1.3.1+cpu-cp37-cp37m-linux_x86_64.whl</code> as your PyTorch package. This provides your version resource with PyTorch 1.3.1 for Python 3.7, built to run on a CPU in Linux.

Use the following command to create your version resource:
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>ack</p>